app/Console/Commands/UserNotificationsCleanup.php

```diff
@@ -59,6 +60,7 @@ public function handle()
                 );
                 $deleted = count($notificationIds);
                 $deletedTotal += $deleted;
+                Datadog::increment(config('datadog-helper.prefix_web').'.notifications_cleanup.user_notifications', $deleted);
```

Comment:
I think you mean
```suggestion
                Datadog::increment(config('datadog-helper.prefix_web').'.notifications_cleanup.user_notifications', 1, null, $deleted);
```
2nd param is sample rate